wait.sh

```diff
@@ -0,0 +1,6 @@
+until nc -z -v -w30 "nftbackend-mysql-1.nftbackend_default" 3306
+do
+  echo "Waiting a second until the database is receiving connections..."
+  # wait for a second before checking again
+  sleep 1
+done
```

Comment:
https://codefather.tech/blog/bash-unexpected-end-of-file/

some Windows new line characters breaks the script(CRLF-LF shit).


```bash
# vim wait.sh -b
until nc -z -v -w30 "nftbackend-mysql-1.nftbackend_default" 3306^M 
do^M 
  echo "Waiting a second until the database is receiving connections..."^M 
  # wait for a second before checking again^M 
  sleep 1^M 
done^M
```